test/influxdb/query_api_test.rb

```diff
@@ -73,6 +88,31 @@ def test_query
     assert_equal 'free', record1.field
   end
 
+  def test_parameterized_query
+    stub_request(:post, 'http://localhost:8086/api/v2/query?org=my-org')
+      .to_return(body: SUCCESS_DATA)
```

Comment:
Please also add check that query params are passed. Something like:

```suggestion
      .with(:body => "{\"query\":\"from(bucket: params.bucketParam) |> range(start:  duration(v: params.startParam)) |> last()\",\"params\":{\"bucketParam\":\"my-bucket\",\"startParam\":\"1970-01-01T00:00:00.000000001Z\"},\"dialect\":{\"header\":true,\"delimiter\":\",\",\"annotations\":[\"datatype\",\"group\",\"default\"],\"commentPrefix\":\"#\",\"dateTimeFormat\":\"RFC3339\"}}")
      .to_return(body: SUCCESS_DATA)
```